### PR TITLE
updating cms report to reflect gross metrics

### PR DESCRIPTION
**Pull request by erubinst**
# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit
)
- [ ] Tests for the changes have been added (for bugfixes / features)

# PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature (requires Feature flag)
- [x] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update version)

# What is the ticket # detailing the issue?

Ticket: https://www.pivotaltracker.com/n/projects/2502930/stories/186740007

# A brief description of the changes

Current behavior: Report only considers latest tax household group when determining medicaid/chip, QHP, and QHP w/ financial assistance eligibility.  This does not reflect gross metrics

New behavior: We now consider all tax household groups in the year when determining medicaid/chip, QHP, and QHP w/ financial assistance eligibility to reflect required gross metrics

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable that is used to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name:

- [ ] DC
- [ ] ME

# Additional Context
Include any additional context that may be relevant to the peer review process.
This update is currently only being made to multi tax household setting as this report is not used for DC

# AppScan CodeSweep Failure
In the event of a failed check on the AppScan CodeSweep step of our GitHub Actions workflow, please review the False Positive protocol outlined here: appscan_codesweep/CODESWEEP_FALSE_POSITIVES_README.MD

Add all required notes to this section if the failure is a suspected false positive.

This false positive matches 2 on the false positive list
1. Vulnerability: `Injection.SQL`
2. Severity: `High`
3. Cause: Code includes an expression with the `where` keyword.
4. False Positive Reasons:
    a. `thhs = family.tax_household_groups.where(:"start_on".gte => Date.new(next_year)).map(&:tax_households).flatten`, here `next_year` is taken from user input that has been cast to an integer.  
    b. The query is called in a script that is run manually and internally.

### CODE REVIEW

**Review by ninavergara605 — APPROVED** [review]

**Review by jayreddy519 — APPROVED** [review]
code changes look good,

please fix github failures before merging. @erubinst

**Review by mdkaraman — APPROVED** [review]
Security Review - Approved

Confirming that the CodeSweep warning matches #2 on the False Positives list for the reasons identified in the description.  The warning does not indicate a credible vulnerability.

**Pull request merged**
